app.js

```diff
@@ -1,157 +1,209 @@
 'use strict';
 
 var correctAnswers = [];
-var wrongAnswers = [];
+// var wrongAnswers = [];
 
+//Wrapped as a function
+function greeting(){
 // Prompt the user to input their name.
 // Store user's name in a variable.
 // Ask the user eight questions and take their answer as input which will be stored in a variable.
 var userName = prompt('What\'s your name?');
 alert('Hello ' + userName + '. Thanks for coming to my site!');
 console.log('The user name is: ' + userName);
+return userName;
+}
+//Calling username function and assigning to userName variable
+var userName = greeting();
```

Comment:
Great use of storing the return value of this function call in the userName variable!